src/app/admin/admin/admin.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-admin',
+  templateUrl: './admin.component.html',
+  styleUrls: ['./admin.component.css']
+})
+export class AdminComponent implements OnInit {
+
+  constructor() { }
```

Comment:
remove if not used

Reply:
Admin component is a to do - This is not the finished version. So I would suggest it to be present.

Reply:
No never would such codes be merged. so i would prefer to have it removed if they are still to do